src/main/java/com/cocodan/triplan/member/repository/MemberRepository.java

```diff
@@ -3,5 +3,19 @@
 import com.cocodan.triplan.member.domain.Member;
 import org.springframework.data.jpa.repository.JpaRepository;
 
+import java.util.List;
+import java.util.Optional;
+
 public interface MemberRepository extends JpaRepository<Member, Long> {
+    @Override
+    Optional<Member> findById(Long aLong);
```

Comment:
```suggestion
    Optional<Member> findById(Long id);
```
aLong 으로 쓰신 이유가 혹시 있으신가요?

Reply:
[답글](https://github.com/prgrms-web-devcourse/Team_Coco_Be/pull/15#discussion_r764627771) 이 댓글에 따라서 Override 부분 제거했습니다!